toolkit/scripts/prerequisites.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash -e
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+prereq_md_path=""
+prereq_json_path=""
+
+function help {
+    echo "Pre-requisite installer. Installs azurelinux pre-requisites for the toolkit."
```

Comment:
Is the aspiration for this script to install the prerequisites, or just scrape out the list of them (for a separate command to install)? (For what it's worth, I'm thrilled to see someone put together an actual honest-to-goodness machine-parseable list of the prerequisites and am supportive of both goals :))

Reply:
This is going to go in another PR, need to polish it up some more. I'm actually going to mark this as a draft for now while I work to stabilize the pipeline side.